File: Data Structures & Algorithms/two-integer-sum/submission-2.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        Dictionary<int,int> hash = new Dictionary<int,int>();
        int[] result = new int[2] {0,0};
        for(int i=0;i<nums.Length;i++)
        { 
            int temp = target - nums[i];
            if(hash.TryGetValue(temp,out int res))
            { 
                result[0]=res;
                result[1]=i;
                
                return result;
            }
            hash[nums[i]] = i;
        }
        return result;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        st = set(nums)
        mx = 0
        c = 1
        for i in range(len(nums)):
            c = 1
            if nums[i]-1 not in st:
                lp = nums[i]
                while lp+1 in st:
                    lp=lp+1
                    c=c+1
                mx = max(mx,c)
        return mx






        